packages/components/src/color-picker/hue-picker.native.js

```diff
@@ -0,0 +1,194 @@
+/**
+ * External dependencies
+ */
+import { Animated, View, PanResponder } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import React, { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import LinearGradient from 'react-native-linear-gradient';
+import styles from './style.native.scss';
```

Comment:
We may be able to drop the `native` reference and rely upon Metro's platform-specific module resolution. I could be wrong. 
```suggestion
import styles from './style.scss';
```